scrapper/scrapper/items.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+# Define here the models for your scraped items
+#
+# See documentation in:
+# http://doc.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+
+
+class ScrapperItem(scrapy.Item):
+    # define the fields for your item here like:
+    # name = scrapy.Field()
+    pass
+
+class menTop(scrapy.Item):
+    #Attributes to scrap
+    name = scrapy.Field()
+    price = scrapy.Field()
+    fit_neck = scrapy.Field()
```

Comment:
So fit_neck would contain a dict like {'small':[10,20], "medium":[20,30]}
These values are on the american eagle website. You can see it if you scroll down on this link: https://www.ae.com/web/browse/product_details.jsp?productId=0153_9141_400&catId=cat6300043